feature_layers/feature-collection-layer-from-portal/README.metadata.json

```
@@ -0,0 +1,28 @@
+{
+    "category": "Feature layers",
+    "description": "Create a feature collection layer from a portal item.",
+    "ignore": false,
+    "images": [
+        "FeatureCollectionLayerFromPortal.png"
+    ],
+    "keywords": [
+        "collection", 
+        "feature collection", 
+        "feature collection layer", 
+        "id", 
+        "item", 
+        "map "notes, 
+        "portal"
+    ],
+    "redirect_from": "/java/latest/sample-code/feature-collection-layer.htm",
+    "relevant_apis": [
+        "FeatureCollection"
+        "FeatureCollectionLayer"
+        "Portal"
+        "PortalItem"
```

These are missing some commas behind them for the json to parse properly:
```suggestion
        "FeatureCollection",
        "FeatureCollectionLayer",
        "Portal",
        "PortalItem"
```